src/patternfly/components/Nav/examples/Navigation.md

```diff
@@ -531,58 +531,29 @@ import './Navigation.css'
 {{> nav--drilldown nav--drilldown--id="level-three-drilldown-example" menu--Drilldown--IsDrilledIn--list-1="true"  menu--Drilldown--IsDrilledIn--list-2="true" menu--Drilldown--menu__content--attribute='style="--pf-v6-c-menu__content--Height: 284px;"' menu--Drilldown--HasCurrentMenuItem="true"}}
 ```
 
-### Nav link text
-When using anything other than a text node for the link text, wrap the link text in an element with `.pf-v6-c-nav__link-text`.
-
+### Nav item icons
 ```hbs isBeta
```

Comment:
Good question - since it's net new and from what I remember, this addition is a part of an eventual enhancement to allow users to collapse the sidebar/vertical nav down to just icons, I think `isBeta` gives us a little wiggle room to make changes we might consider breaking otherwise as we move to being able to collapse the sidebar that way. But this is simple enough, I don't see marking it as beta as necessary either. WDYT @andrew-ronaldson @srambach?